docs/content/0.Config/5.Automate/1.Usage.md

```diff
@@ -1,3 +1,44 @@
+---
+title: Usage
+---
+
 # Automate
 
-Not yet documented ...
\ No newline at end of file
+The "Automate" feature can be used to **periodically request one or more PID**s and post the results to an **MQTT server**. Therefore, this feature requires [MQTT configured in WiCAN](../MQTT.md).
+
+Configuration of "Automate" is done on the "Automate" tab.  
+
+## Vehicle Profiles
+The "Automate" feature automatically downloads **vehicle profiles** from GitHub containing some popular PIDs (e.g. SoC, SoH). These vehicle profiles are car vendor and/or model specific. There is a growing [list of supported vehicles](2.Supported_Vehicles.md).
+
+If WiCAN cannot download vehicles profiles for any reason you man also [download in manually](https://github.com/meatpiHQ/wican-fw/blob/main/vehicle_profiles.json) and press the `Choose File` button to select the file downloaded.
+
+In order to use vehicle profiles `Vehicle Specific` has to be set to `Enable`.
+
+If vehicles profiles are available to WiCAN the appropriate `Vehicle Model` should be selected from the drop-down box. Use the reload button next to it to trigger the download of the most up-to-date vehicle profiles.
+
+In `Cycle Time(ms)` the interval is configured after which the PIDs, contained in the vehicle profile, are requested. 
+
+The results are posted to the MQTT topic specified in `Destination Topic`. 
+
+![image](/config/automate/automate.png)
+
+
+## Custom PIDs
```

Comment:
at the moment Custom PIDs are only supported over mqtt, i will extend it later to be supported in the integration. So mqtt has to be enabled to support this feature

Reply:
This is stated already in the common section on top of the page now.